backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -1126,31 +1104,6 @@ def forward(self, x):
         model_name = "ptq_qnn_pixel_shuffle_model"
         save_model_and_expected_output(instance, buffer, example_inputs, model_name)
 
-    def test_qnn_backend_ptq_home_made_pixel_shuffle(self):
```

Comment:
same here. Why is this test removed?

Reply:
The reason is similar to the change of EDSR model as aforementioned, thanks.